Makefile

```diff
@@ -129,7 +129,7 @@ evaluate-clueweb12: install
 	./src/main/bash/run-evaluation-report.sh -c CLUEWEB12 --threshold 0.84|tee results/clueweb12-evaluation.jsonl
 
 install: install-third-party
-	mvn clean install
+	./gradlew clean publishToMavenLocal
```

Comment:
The `publishToMavenLocal` command builds the Gradle project and publishes the JAR artifact to the local Maven repo, effectively what `mvn install` did.